src/main/java/alfio/controller/support/TemplateProcessor.java

```diff
@@ -62,7 +63,7 @@ public static PartialTicketTextGenerator buildPartialEmail(Event event,
                                                                Map<String, Object> additionalOptions) {
         return ticket -> {
             Map<String, Object> model = TemplateResource.buildModelForTicketEmail(organization, event, ticketReservation, baseUrl, ticketURL, calendarURL, ticket, category, additionalOptions);
-            return templateManager.renderTemplate(event, event.getIsOnline() ? TemplateResource.TICKET_EMAIL_FOR_ONLINE_EVENT : TemplateResource.TICKET_EMAIL, model, language);
+            return templateManager.renderTemplate(event, EventUtil.isAccessOnline(category, event) ? TemplateResource.TICKET_EMAIL_FOR_ONLINE_EVENT : TemplateResource.TICKET_EMAIL, model, language);
```

Comment:
it is possible to do event.isAccessOnline(category) instead of the helper?

Can be a default method in `EventCheckInInfo`

Reply:
would be cool indeed. But that would be a potential issue with implementations being able to override that method. 
At the moment there's no way of having a `default final` method in interfaces (not sure it would make sense though)